src/app/Main/Body/QueryPage/Results/TableResults/TableContent-RowWithColSums.component.tsx

```diff
@@ -63,12 +62,16 @@ export function TableContentRowWithColSumComponent(props: {
         </TableCell>,
     ];
 
+    let totalColSum = 0
+
     props.headerValues.forEach((headerValue) => {
         let colSum: number | string = calculateColSum(
             props.tableData,
             headerValue
         );
 
+        totalColSum += colSum

```

Comment:
You use colSum as a container for number and string representation, so adding here is not very fine. I suggest to use an additional colSumString variable as you've done for the totalSum below.

Reply:
Done.